### DISCUSSION TITLE
新要求需要每页都有页眉

### DISCUSSION THREAD

**Discussion by GengchenXU** [xjtu-blacksmith/easymcm#16] [Q&A]
![image](https://user-images.githubusercontent.com/56543168/153829480-f286e0e1-d971-477a-bdc2-fb86cf06b370.png)
应该如何修改？

**Comment by GengchenXU** [comment]
我目前是把pagestyle{empty}改为fancy
![image](https://user-images.githubusercontent.com/56543168/153835406-09a37797-0535-47d3-a6ca-89c0fe7c3598.png)
同时
![image](https://user-images.githubusercontent.com/56543168/153835611-b6e6fbef-cc0d-4cec-a456-f305183eef39.png)
把-5pc改为-3pc

**Reply by xjtu-blacksmith** [reply]
您好！检查了目前的 instruction，确有此条，但对您所指出的修改有不同看法。要求中说的是 solution 的每一页都要有页眉，但 summary sheet 似乎并不被视作 solution 的一部分；这一点，您可以从 COMAP 官网下载今年的 [Word](https://www.comap.com/undergraduate/contests/mcm/flyer/MCM-ICM_Summary.docx) 或 [LaTeX](https://www.comap.com/undergraduate/contests/mcm/flyer/MCM-ICM_Summary.tex) summary sheet 模板中自行检查，这两份 summary sheet 模板中均未在页眉有任何标注。此外，从您所截的 "During the Contest – Preparing a Solution Document" 之第 4 点中，也可发现：目录、参考文献、附录等均强调是计入页数的，但并未提到过 summary sheet 要计进页数，由此也可说明 summary sheet 并不属于 solution。

另外，您提到这个页眉要求是「新要求」，我印象里似乎不是，去年的时候就有讨论（参见 #9）；当时我没有及时确认，之后在 v5.2.1 中进行了更正，给目录加上了页眉。今年 instruction 中也有第 III 节 "Changes for 2022"，里面并未提到页眉上的标记是新增的要求。

以上意见，仅供参考。我自己参加美赛已是 2018 年的事情，无法确认自己的推测是否准确，还请您和其他用户仔细斟酌、共同讨论。感谢反馈！
